src/api_client/api_client.js

```diff
@@ -69,29 +59,36 @@ export default class APIClient {
 	}
 
 	static createMainnetAPIClient(options) {
-		return new APIClient(MAINNET_NODES, MAINNET_NETHASH, options);
+		return new APIClient(
+			MAINNET_NODES,
+			Object.assign({}, options, { nethash: MAINNET_NETHASH }),
```

Comment:
Is this definitely the right order? Could there be a use case where somebody wants to use this with a custom nethash? (In which case `Object.assign({}, { nethash: MAINNET_NETHASH }, options)` would be better.)